### ISSUE TITLE
[Bug]: 2.2.14 Using the official website docker deployment process to test the performance is very slow than 2.2.10

### ISSUE THREAD

**Issue by become-nice** [milvus-io/milvus#28564]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Environment

```markdown
- Milvus version: 2.2.10 2.2.14
- Deployment mode(standalone or cluster): standalone
- MQ type(rocksmq, pulsar or kafka):    
- SDK version(e.g. pymilvus v2.0.0rc2): 2.2.15
- OS(Ubuntu or CentOS): Ubuntu
- CPU/Memory: 16c
- GPU: 
- Others:
```

### Current Behavior

&emsp;&emsp;When I started a standalone milvus instance according to the [official website deployment process](https://milvus.io/docs/v2.2.x/install_standalone-docker.md) and tested it with the vectordbbench tool, I found that on the same machine, the performance of 2.2.10 was 3 times higher than that of 2.2.14. What is the reason for this?
**2.2.10**
* Download config file
```
wget https://github.com/milvus-io/milvus/releases/download/v2.2.10/milvus-standalone-docker-compose.yml -O docker-compose.yml
```
* Start standalone instance
![WechatIMG1998](https://github.com/milvus-io/milvus/assets/56624819/88d418cd-f89e-4da0-aaf2-e5393a6316fe)
* Show test result
![WechatIMG1999](https://github.com/milvus-io/milvus/assets/56624819/3f511a9b-ee44-4451-a7fd-8d912f52c8b9)

**2.2.14**
* Donwload config file
```
wget https://github.com/milvus-io/milvus/releases/download/v2.2.14/milvus-standalone-docker-compose.yml -O docker-compose.yml
```
* Start standalone instance
![WechatIMG1996](https://github.com/milvus-io/milvus/assets/56624819/e8806964-39b3-46fc-9c1f-1d411650d1c2)
* Show test result
![WechatIMG1997](https://github.com/milvus-io/milvus/assets/56624819/c27d2b75-2b96-454f-9109-90a719020177)

**Comment by become-nice** [comment]
![Uploading WeChat82b2c66fadfbfa3297c50e79dc410e03.png…]()

**Comment by yanliang567** [comment]
I think there are BF searches in the test2, @liliu-z  could you please help to identify that and share some suggestion?

/assign @liliu-z 
/unassign

**Comment by become-nice** [comment]
In actual use, version 2.2.14 is also significantly slower than 2.2.10.

**Comment by yanliang567** [comment]
any updates for this issue?

**Comment by xiaofan-luan** [comment]
I believe this is due to the index policy change and should be fixed by 2.2.16
please check
